libgraph/include/katana/PropertyGraph.h

```diff
@@ -266,10 +266,16 @@ class KATANA_EXPORT PropertyGraph {
       const katana::RDGLoadOptions& opts = katana::RDGLoadOptions());
 
   /// Make a property graph from topology
+  // [deprecated("please provide a storage prefix")]
```

Comment:
A boring but useful followup task would be to add the `[[deprecated]]` attribute: See https://thek-teamworkspace.slack.com/archives/C02C1F8C9DL/p1644535228882339